.github/workflows/debos.yml

```diff
@@ -46,66 +48,29 @@ jobs:
       # avoid an automated update while we're building
       - name: Update OS packages
         run: |
-          set -x
-          sudo apt update
-          sudo apt -y upgrade
-          sudo apt -y full-upgrade
-
-      # this is the default in our self-hosted runners
-      - name: Make sure Incus is setup
-        run: |
-          set -x
-          sudo apt -y install incus
-          sudo incus admin init --auto
-
-      # create a fresh container build environment to decouple the build
-      # operating system from the github runner one; install debos
-      - name: Setup build environment
-        run: |
-           set -x
-           # privileged container as debos will use mounts
-           sudo incus init "${INCUS_IMAGE}" "${INCUS_NAME}" \
-               -c security.privileged=true -c security.nesting=true
-           sudo incus start "${INCUS_NAME}"
-           # wait for network to be up (prior to running apt)
-           sudo incus exec "${INCUS_NAME}" \
-              /usr/lib/systemd/systemd-networkd-wait-online
-           (
-               # these commands are run inside the container
-               cat <<EOF
-               apt update
-               apt -y upgrade
-               apt -y full-upgrade
-               apt -y install debos
-           EOF
-           ) | sudo incus exec "${INCUS_NAME}" -- sh
+          set -ex
```

Comment:
Rest of script only uses -x; I was under the impression github jobs were operating under set -e already, so perhaps we should stick to set -x at the top of each job